.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/sakai/
+    docker:
+      - image: maven:3.5
+    steps:
+      - checkout
+      - restore_cache:
+          key: MAVEN
+      - run:
+          name: Maven Install
+          command: mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+      - save_cache:
+          key: MAVEN
+          paths:
+            - ~/.m2/
```

Comment:
It might be worth just removing all the org.sakaiproject SNAPSHOTS before making the cache so that the build catches changes that depend on artifacts no longer in the build. In travis this is done in the `before_cache` step with `find $HOME/.m2/repository/org/sakaiproject -name \*-SNAPSHOT.\*  -delete`